File: app/src/main/java/com/example/db/adapter/StudentAdapter.java
package com.example.db.adapter;

import android.graphics.Color;

import com.chad.library.adapter.base.BaseQuickAdapter;
import com.chad.library.adapter.base.module.LoadMoreModule;
import com.chad.library.adapter.base.viewholder.BaseViewHolder;
import com.example.db.R;
import com.example.db.entity.Student;

import java.util.List;

public class StudentAdapter extends BaseQuickAdapter<Student, BaseViewHolder> implements LoadMoreModule {

    public StudentAdapter(List<Student> data) {
        super(R.layout.item_rv_student, data);
        addChildClickViewIds(R.id.tv_del);
    }

    @Override
    protected void convert(BaseViewHolder helper, final Student item) {
        helper.setText(R.id.tv_id, String.valueOf(item.id))
                .setText(R.id.tv_name, item.name)
                .setText(R.id.tv_age, String.valueOf(item.age))
                .setText(R.id.tv_score, String.valueOf(item.score));
        if (item.name.startsWith("ZH")) {
            helper.setTextColor(R.id.tv_name, Color.GRAY);
        } else {
            helper.setTextColor(R.id.tv_name, Color.BLACK);
        }

    }

}

File: app/src/main/java/com/example/db/adapter/StudentAdapter.kt
package com.example.db.adapter

import android.graphics.Color
import com.chad.library.adapter.base.BaseQuickAdapter
import com.chad.library.adapter.base.module.LoadMoreModule
import com.chad.library.adapter.base.viewholder.BaseViewHolder
import com.example.db.R
import com.example.db.entity.Student

class StudentAdapter(data: List<Student>) : BaseQuickAdapter<Student, BaseViewHolder>(R.layout.item_rv_student), LoadMoreModule {

    override fun convert(holder: BaseViewHolder, item: Student) {
        holder.setText(R.id.tv_id, item.id.toString())
                .setText(R.id.tv_name, item.name)
                .setTextColor(R.id.tv_name, if (item.name.startsWith("ZH")) Color.GRAY else Color.BLACK)
                .setText(R.id.tv_age, item.age.toString())
                .setText(R.id.tv_score, item.score.toString())
    }

}